xml/deploy_additional.xml

```diff
@@ -300,6 +300,53 @@
       </listitem>
      </itemizedlist>
     </note>
+    <important>
+     <title>Incorrectly reported image sizes and the <option>emulate_legacy_capacity</option> option</title>
+     <para>
+      A bug in older kernels caused that an RBD target reported an image size
+      one block (512 bytes) larger than it actually was. Therefore, the LUN
+      could write one additional block of the image. To keep the image data
+      consistent, new kernels support the old behavior by default unless you
+      set <option>emulate_legacy_capacity</option> attribute to 0 for a target.
+     </para>
+     <para>
+      We recommend disabling the old behavior for newly created targets. If you
+      plan to enable the <option>object-map</option> feature for RBD images,
+      disabling <option>emulate_legacy_capacity</option> is required. Use the
+      &gwcli; shell to disable it:
+     </para>
+<screen>
+&prompt.root;&gwcli;
+&prompt.gwcli;cd /disks/<replaceable>POOL</replaceable>/<replaceable>IMAGE</replaceable>
+&prompt.gwcli;reconfigure emulate_legacy_capacity 0
+</screen>
+     <tip>
+      <para>
+       You can also disable the <option>emulate_legacy_capacity</option> option
+       in the &dashboard; when creating or editing the RBD target.
+      </para>
```

Comment:
It probably makes sense to state that the dashboard can be used for this before documenting the `gwcli` process, as the former should be a little more user friendly.

Also, I think it'd be good to have a warning here that the above steps should only be taken for new images which have *not* been exposed via iSCSI. Existing images require the rbd image to be resized...